src/Infrastructure/TemplateBasedMailer.php

```diff
@@ -36,4 +36,17 @@ public function sendMail( EmailAddress $recipient, array $templateArguments = []
 		);
 	}
 
+	/**
+	 * @throws \RuntimeException
+	 */
+	public function sendMailToOperator( EmailAddress $replyToAddress, array $templateArguments = [] ) {
```

Comment:
What made me add this method is the lack of support for `Messenger::sendMessageToOperator()` in this class. We want to use a template for the message content, but use the fixed operator's address as the recipient and add a `Reply-To` header for convenience.

We could instead introduce another parameter for the value of the `Reply-To` header. That would force us to make all parameters default to `null` (since `Messenger::sendMessageToOperator()` does not expect the recipient to be passed). Or we could pass a flag to let the method know what the passed e-mail address should be used as.

But I don't prefer any of the above approaches to adding another method. What do you think?